indico/web/templates/forms/multiple_items_widget.html

```diff
@@ -89,20 +90,35 @@
                 e.preventDefault();
                 var row = $(this).closest('tr');
                 var item = {};
+                var invalidInput = false;
                 if (uuidField && row.data('uuid')) {
                     item[uuidField] = row.data('uuid');
                 }
                 columns.forEach(function(col, i) {
-                    item[col[0]] = row.find('input').eq(i).val();
+                    var inputField = row.find('.js-table-input').eq(i);
+                    var value = inputField.val();
+                    if (value || !inputField.data('required')){
```

Comment:
I think it's more readable if you reverse this, i.e. `!value && inputField.data('required')` - also, I'd `.trim()` the value